code/game/objects/structures/window.dm

```diff
@@ -595,6 +595,30 @@
 /obj/structure/window/plastitanium/unanchored
 	anchored = FALSE
 
+//pirate ship windows
+/obj/structure/window/plastitaniumpirate
```

Comment:
Make this a subtype of `/obj/structure/window/plastitanium` and remove any duplicate variable statement on this path.